druid-shell/src/text.rs

```diff
@@ -531,6 +549,38 @@ pub fn simulate_input<H: WinHandler + ?Sized>(
             };
             input_handler.handle_action(action);
         }
+        KbKey::Home => {
+            let movement = Movement::ParagraphStart;
+            if event.mods.shift() {
+                input_handler.handle_action(Action::MoveSelecting(movement));
+            } else {
+                input_handler.handle_action(Action::Move(movement));
+            }
+        }
```

Comment:
this doesn't looks correct to me, home moves to the start of line, and ctrl-home to the start of document. 
(similarly for end)